README.md

```diff
@@ -204,6 +205,9 @@ config {
   - `unveil` - (optional) - A list of additional filesystem paths to provide
   access to the task (requires `unveil_by_task` in plugin config).
 
+  - `oom_score_adj` - (optional) - The likelihood of the task being OOM killed,
+  must be a positive integer.
```

Comment:
should specify the default value

Reply:
[fc420ad](https://github.com/hashicorp/nomad-driver-exec2/pull/40/commits/fc420ad6f5ff45c618841f37a78ed0f89488c50f)